src/main/java/com/dongtronic/diabot/commands/nightscout/NightscoutSetDisplayCommand.kt

```diff
@@ -10,7 +10,8 @@ import org.slf4j.LoggerFactory
 
 class NightscoutSetDisplayCommand(category: Command.Category, parent: Command?) : DiabotCommand(category, parent) {
     companion object {
-        val validOptions = arrayOf("none", "title", "trend", "cob", "iob", "avatar")
+        val enabledOptions = arrayOf("title", "trend", "cob", "iob", "avatar")
+        val validOptions = enabledOptions.plus(arrayOf("simplified", "none"))
```

Comment:
Please rename `simplified` to `simple`, or somehow add the ability to create aliases.

Reply:
Yeah, that looks better. I'll just change it to simple and maybe do the aliases later.